recipe/meta.yaml

```diff
@@ -24,24 +24,25 @@ requirements:
     - pip
     - python >=3.8
   run:
-    - python >=3.8
     - dask >=2022.3.0,<2022.4.1
-    - pandas >=1.0.0
+    - fastapi >=0.61.1
     - jpype1 >=1.0.2
     - openjdk >=8
-    - fastapi >=0.61.1
-    - uvicorn >=0.11.3
-    - tzlocal >=2.1
+    - nest-asyncio
+    - pandas >=1.0.0
     - prompt-toolkit
     - pygments
-    - nest-asyncio
+    - python >=3.8
     - tabulate
+    - tzlocal >=2.1
+    - uvicorn >=0.11.3
```

Comment:
@jakirkham for context, we are considering reordering the packages in the way generated by grayskull to hopefully make package diffs easier to parse moving forward - LMK if that makes sense to you 🙂

Reply:
Yeah that's what I gathered. Seems ok

Also regarding dependency ordering, there are a few relevant upstream issues: 
* https://github.com/conda-incubator/grayskull/issues/259
* https://github.com/conda-incubator/grayskull/issues/288

Reply:
Cool, thanks for sharing - was interested in how those issues were being recieved